src/components/Root.tsx

```diff
@@ -40,7 +40,7 @@ function render(props: RootLayoutProps) {
     />
   )
   const Test = () => <div>REACT-FEATURE-TOGGLE-TEST</div>
-  const TestFeature = configureFeature(Main, 'foo', Test)
+  const TestFeature = configureFeature(Main)('foo')(Test)
```

Comment:
Just curious — is this needed by the new version or just a preference?

Reply:
It's required by the new version. We were using auto curry from lodash before, but I have gotten tired of the auto curry function
- breaking (IntelliSense) code hints
- requiring extra unit tests for all forms of partial application due to weird issues it can cause.
- requiring extra overload types

I think the small difference in syntax is worth avoiding those things, I am trying to remove auto curry from all my code if possible, and I am finding I actually don't mind the extra parenthesis :D